sysutils/AutoRaise/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sbmpost AutoRaise 1.6 v
+revision            0
+
+categories          sysutils
+platforms           macosx
```

Comment:
Shouldn't this be `platforms darwin`?

Reply:
Maybe, but since I do not have open-source Darwin to verify and the docs say `platforms` is "not interpreted in any way by MacPorts at this time; it is purely informational for users.", I'd stick with the confirmed "macos", rather than guessing or hoping "darwin" would also work.
If you can confirm AutoRaise runs on pure Darwin as well, please go ahead and update the Portfile (it's marked `openmaintainer` after all)

Reply:
The Portfile reference has the following:

> Ports for software that does not require macOS-specific features should use darwin. Most ports use this value on the presumption that they would work on Pure Darwin, even if that has not been attempted. Ports for software that is known to require macOS-specific features should set this to macosx.

I have no idea which part of macOS is open-sourced and how to determine if a port requires macOS-specific features. If you know the details, please feel free to change the platform to darwin any time.

Reply:
I guess this is now obsolete with the switch to xcode...